dashboard/serializers.py

```diff
@@ -47,5 +50,6 @@ def serialize(cls, program_enrollment):
             'id': program.id,
             'enrollments': program_cached_enrollments,
             'certificates': program_cached_certificates,
-            'grade_average': cls.calculate_certificate_grade_average(program_cached_certificates)
+            'grade_average': cls.calculate_certificate_grade_average(program_cached_certificates),
+            'non_learner': has_role(user, NON_LEARNERS)
```

Comment:
You will need also to write another test to cover the case that the user is not Staff of the program and in this case it appears in the search result